demos/datagrid/script.js

```diff
@@ -53,9 +52,9 @@ require(['lib/husky'], function(Husky) {
                     matchings: 'http://husky.lo:7878/admin/api/datagrid/fields',
                     selectedCounter: true,
                     selectedCounterText: 'selected elements',
-                    clickCallback: function(item, data) {
+                    /*clickCallback: function(item, data) {
                         app.logger.log('Husky.Ui.DataGrid item click: ' + item, data);
-                    },
+                    },*/
```

Comment:
Remove this code if it is not required. But I would leave it there, or did you have a reason to remove it?